pkg/provider/provider.go

```diff
@@ -66,3 +68,30 @@ func GetProviderDetails() []Details {
 		},
 	}
 }
+
```

Comment:
Not sure what past me was thinking with the string concatenation 🤦‍♂️ 

This function aims to combine the 2 URLs safely. So far the API URL doesn't have any query string params but I don't think merging the query strings does any harm.